Dnn.AdminExperience/Library/Dnn.PersonaBar.Library/DTO/UserSettings.cs

```diff
@@ -34,7 +34,7 @@ namespace Dnn.PersonaBar.Library.DTO
     /// Persona Bar Settings For User.
     /// </summary>
     [DataContract]
-    public class UserSettings : Dictionary<string, object>, IXmlSerializable
+       public class UserSettings : Dictionary<string, object>, IXmlSerializable
```

Comment:
Can this extra whitespace be removed?

Reply:
I can not commit my changes after rebasing into origin/release 9.4.x?